pom.xml

```diff
@@ -34,7 +34,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>8-jre</docker.tag>
```

Comment:
Can we leave the issue of build ids / timestamps to a different Issue/PR, as it may be a little complex, or at least need careful though with how it is going to interact with version and CI.